src/PointwiseFunctions/AnalyticData/SelfForce/GeneralRelativity/CircularOrbitCoeffs.hpp

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/// \file
+/// These functions are generated by Mathematica. They compute the terms in the
+/// general relativistic self-force equations for a circular orbit in Kerr
+/// spacetime. See CircularOrbit.hpp for details.
+
+namespace GrSelfForce::detail {
+
+int getAreal(int m, double a, double om, double r, double th,
+             double Areal[10][10]);
+
+int getAimag(int m, double a, double om, double r, double th,
+             double Aimag[10][10]);
+
+int getBreal(int m, double a, double om, double r, double th,
+             double Breal[10][10]);
+
+int getBimag(int m, double a, double om, double r, double th,
+             double Bimag[10][10]);
+
+int getCreal(int m, double a, double om, double r, double th,
+             double Creal[10][10]);
+
+int getCimag(int m, double a, double om, double r, double th,
+             double Cimag[10][10]);
```

Comment:
Can you replace these with `std::array<std::array<double, 10>, 10>`? I assume you don't format the cpp file so that if the equations are change, diffs are somewhat remotely close to readable? Then again, formatting might actually make diffs more readable, since the new copy-pasted code would also get formatted and then git would diff only changed lines. I'm inclined to lean towards "formatting is better"

Reply:
Mostly done. Just formatting of the file `CircularOrbitCoeffs.cpp` would increase the file size by quite a lot because of all the extra whitespace, so I didn't do it.